### PR TITLE
Add SXGs to /optimize-lcp

### PR DESCRIPTION
**Pull request by khempenius**
cc @twifkak

### CODE REVIEW

File: src/site/content/en/blog/optimize-lcp/index.md [hunk 1]
```diff
@@ -168,6 +169,15 @@ Learn more by reading [Establish network connections early to improve perceived
...
 
+### Use signed exchanges (SXGs)
+
+[Signed exchanges (SXGs)](https://web.dev/signed-exchanges) are a delivery mechanism that allow for
+faster user experiences by providing content in an easily cacheable format.
+Specifically, [Google Search](https://developers.google.com/search/docs/advanced/experience/signed-exchange)
+will cache and sometimes prefetch SXGs. For sites that receive a large portion
+of their traffic from Google Search, SXGs can be an important tool for improving
+LCP. For more information, see [Signed
+Exchanges](https://web.dev/signed-exchanges).
```
Comment (kaycebasques) [inline src/site/content/en/blog/optimize-lcp/index.md:180]:
```suggestion
Exchanges](/signed-exchanges).

```

(Re: the extra space) Some Markdown parsers need a space here to detect the new section heading

**Review by kaycebasques — CHANGES_REQUESTED** [review]
@khempenius while you're here would you mind fixing those Markdown linting problems that the bot detected: https://github.com/GoogleChrome/web.dev/pull/5143#issuecomment-824482230

**Comment by kaycebasques** [comment]
@khempenius you might be working on this now but just noting here that you've got a merge conflict

**Comment by twifkak** [comment]
Was the merge conflict addressed by the latest commits?

**Comment by khempenius** [comment]
@kaycebasques can you take a look at this? This is the first PR that I've run into pullapprove. I believe the merge conflict has been resolved.

**Review by kaycebasques — APPROVED** [review]

**Comment by kaycebasques** [comment]
@khempenius you can merge when you're ready

**Pull request merged**